python/grass/jupyter/interactivemap.py

```diff
@@ -347,13 +353,160 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def draw_computational_region(self):
+        """
+        Allow Users to draw the computational region and modify it.
+        """
+        import ipywidgets as widgets
+        from ipyleaflet import Rectangle, WidgetControl
+
+        region_mode_button = widgets.ToggleButton(
+            icon="square",
```

Comment:
Let's use an icon that looks like outline of a rectangle (not solid square)

Reply:
How about [this?](https://fontawesome.com/v4/icon/square-o)